src/wng_building/station_river_creator.py

```diff
@@ -1,3 +1,124 @@
+import copy
+
+from src.data_handling.data_handler import DataHandler
+
+
 class StationRiverCreator:
```

Comment:
Itt is az a tippem, hogy az itteni metódusokat csak egyszer akarjuk futtatni egy betöltött adatra - kellene egy `RiverDataInterface` osztály, ami csak a tagváltozókat (`stations`, `river`, `completed_river`) tárolja, mert úgyis csak ezt akarjuk átadni a WNG építőnek.